frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_node_widget.dart

```diff
@@ -333,12 +364,10 @@ class _SmartEditInputState extends State<_SmartEditInput> {
   }
 
   Future<void> _requestCompletions() async {
+    final getIt = GetIt.instance;
     final result = await UserBackendService.getCurrentUserProfile();
     return result.fold((l) async {
-      final openAIRepository = HttpOpenAIRepository(
-        client: client,
-        apiKey: l.openaiKey,
-      );
+      final openAIRepository = await getIt.getAsync<OpenAIRepository>() as HttpOpenAIRepository;
```

Comment:
```suggestion
      final openAIRepository = getIt<OpenAIRepository>() as HttpOpenAIRepository;
```
Will this not suffice?